addon_test.template.sh

```diff
@@ -10,14 +10,15 @@ ${buildout:directory}/bin/mrbob -O ${:addon_name} -n bobtemplates:plone_addon --
 cd ${buildout:directory}/${:addon_name}
 virtualenv ${buildout:directory}/${:addon_name}
 ${buildout:directory}/${:addon_name}/bin/python bootstrap-buildout.py --setuptools-version=8.3
-${buildout:directory}/${:addon_name}/bin/buildout
+${buildout:directory}/${:addon_name}/bin/buildout code-analysis:return-status-codes=True
```

Comment:
👍 so now that you found it, would you mind removing all the debug stuff? we certainly don't want to keep that around :-)